tcms/testruns/static/testruns/js/get.js

```diff
@@ -120,15 +120,24 @@ function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
             const listGroupItem = $(`.test-execution-${testExecution.id}`)
             listGroupItem.find('.test-execution-priority').html(testCase.priority)
             listGroupItem.find('.test-execution-category').html(testCase.category)
+            listGroupItem.find('.test-execution-text').html(testCase.text)
+            listGroupItem.find('.test-execution-notes').append(testCase.notes)
 
             const isAutomatedElement = listGroupItem.find('.test-execution-automated')
             const isAutomatedIcon = testCase.is_automated ? 'fa-cog' : 'fa-thumbs-up'
             const isAutomatedAttr = testCase.is_automated ? isAutomatedElement.data('automated') : isAutomatedElement.data('manual')
             isAutomatedElement.addClass(isAutomatedIcon)
             isAutomatedElement.attr('title', isAutomatedAttr)
 
-            jsonRPC('TestExecution.get_links', { 'execution_id': testExecution.id, 'is_defect': true }, bugs => {
-                listGroupItem.find('.test-execution-bugs-count').html(bugs.length)
+            jsonRPC('TestExecution.get_links', { 'execution_id': testExecution.id }, links => {
+                const bugCount = links.filter(link => link.is_defect).length;
+                listGroupItem.find('.test-execution-bugs-count').html(bugCount)
+
+                const ul = listGroupItem.find('.test-execution-hyperlinks')
+                links.forEach(link => {
+                    const icon = link.is_defect ? `<span class="fa fa-bug"></span>` : ''
+                    ul.append(`<li>${icon} <a href="${link.url}">${link.name || link.url}</a></li>`)
+                })
```

Comment:
No HTML in JavaScript. Use `<template><li>...</li></template>` and  render with the appropriate attributes for each data entry coming from RPC.